### PR TITLE
acknowledgements page added

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
**Description**
Added a new page to the website - Acknowledgements. 
Added links to the new page in the nav bar and footer
The page is done following the mock up. Figma link of it is provided below.

**Mock Up**
[Mock Up Link](https://www.figma.com/file/fXh58MfxEDh1ssgxQyPb5f/Overture_Refresh?type=design&node-id=686%3A1494&t=0EMBBJkQeLS1dj1Y-1)

**Screenshots (Updated June 22nd):**

- Mock-Up Screenshot
- NavBar and Upper portion of Acknowledgements Page of the Mock-Up
Screenshot 2023-06-22 at 10 56 52 AM

- Lower portion of the Acknowledgements Page and Footer of the Mock-Up
Screenshot 2023-06-22 at 10 57 06 AM

- Dev Screenshot

- NavBar and Upper portion of Acknowledgements Page of Dev
Screenshot 2023-06-22 at 10 57 19 AM

- Lower portion of the Acknowledgements Page and Footer of the Dev
Screenshot 2023-06-22 at 10 57 32 AM

### CODE REVIEW

**Comment by daniel-cy-lu** [comment]
Having a bit of trouble connecting to Zenhub. Trying to fix it.

**Comment by MitchellShiell** [comment]
@daniel-cy-lu thanks for posting this and laying it all out. Have a look at the mock-ups you've linked, notably the top nav, hero and headers.

File: src/pages/acknowledgements/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+import Helmet from 'react-helmet';
+import { Hero, H2, H3, MarketingSection } from 'components';
+import './styles.scss';
+
+export default function AcknowledgementsPage() {
+  return (
+    <main className="AcknowlegementsPage">
+      {/* Metadata */}
+      <Helmet>
+        <title>Overture Aknowlegements</title>
```
Comment (samrichca) [inline src/pages/acknowledgements/index.js:11]: Aknowlegements should be Acknowledgements

File: src/pages/acknowledgements/styles.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
+@import '../../styles/colors';
```
Comment (samrichca) [inline src/pages/acknowledgements/styles.scss:1]: add copyright

File: src/pages/acknowledgements/index.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        />
+        <meta
+          name="keywords"
+          content="Overture, data science software, bioinformatics software, open-source software, cancer research, academic collaborations, grant co-applicant, software consulting, project architecture, migration, custom development, scalability, technical support, troubleshooting, Ontario Institute for Cancer Research, OICR, Canarie, DMS Command Line Interface, The National Cancer Institutes Informatics Technology for Cancer Research Program, NCI ITCR, Overture DMS, GA4GH passport system"
+        />
+      </Helmet>
+
+      {/* HERO */}
+      <Hero
+        title="Acknowlegements"
```
Comment (samrichca) [inline src/pages/acknowledgements/index.js:24]: missing a D in acknowledgements

File: src/pages/acknowledgements/styles.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    padding: 80px 0;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .header {
+    display: flex;
+    justify-content: center;
+
+    @media (max-width: 1024px) {
```
Comment (samrichca) [inline src/pages/acknowledgements/styles.scss:29]: use the variable $tablet for 1024px

File: src/pages/acknowledgements/index.js (hunk 3 above)
Comment (daniel-cy-lu) [reply src/pages/acknowledgements/index.js:24]: omg.. I will copy and paste instead of spelling this work out from now on

**Comment by daniel-cy-lu** [comment]
**Update:** There are a couple UI related changes I need to make after taking to Mitchell on Thurs the 15th. The idea is to get this page as closing identical to the mock up as possible. Thank you for reviewing the PR. I will let you know when the changes are done and `git push` @samrichca @MitchellShiell

**Comment by daniel-cy-lu** [comment]
After comments from Patrick and Mitchell to make final product as identical as possible to the mock up, I made a few changes. This update includes the navbar, footer, and Acknowledgements Page. Colour, font weight, font size are ALL identical to the mock up. Exception due to technical reasons, Lato font only has 700 or 900 but not 800. So for the portion in the mock up that uses font weight 800, 900 is used instead. The visual result is excellent. Components in Typography such as subtitle style, paragraph style and list style are created to be re-used in the later pages. Screenshot in this ticket is updated. @MitchellShiell @samrichca @patrickdsantos

**Pull request closed without merging**